addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -560,6 +561,7 @@ QUnit.module("Fields", (hooks) => {
         await changeType(target, "integer");
         await closePopover(target);
 
+        await editValue("0", "0");
```

Comment:
can we also add a python test for both `value` and `default` key ?
in `test_properties_field_integer_float_boolean`, write zero and check that we read zero and not false